contracts/FlowCallbackScheduler.cdc

```diff
@@ -614,10 +691,18 @@ access(all) contract FlowCallbackScheduler {
                 let mediumPriorityIDs: [UInt64] = []
 
                 for id in callbackIDs.keys {
-                    if self.getPriority(id: id) == Priority.High {
-                        highPriorityIDs.append(id)
+                    let callback = self.borrowCallback(id: id)!
+
+                    // If any callbacks failed to execute properly because of an error,
+                    // mark them as executed
+                    if callback.status == Status.Processed {
+                        self.finalizeCallback(callback: callback, status: Status.Executed)
+                        continue
                     }
```

Comment:
This addition is what I am curious about. Is it okay to clean the failed callbacks here in `process()`, or does it need to be an explicit fvm transaction for each one?

Reply:
Sure, we need to discuss it more, but we can't just not handle failed transactions at the moment or the contract will get in a weird state with all the transactions that never leave the `Processed` state. I think we should either keep this or the manual way to mark failed callbacks as executed and create an issue to revisit it in the future to decide on a final solution.